### DISCUSSION TITLE
Crashlytics reports show "Missing BuildId" instead of symbols in the call stack

### DISCUSSION THREAD

**Discussion by VioletGiraffe** [firebase/firebase-android-sdk#6153] [Q&A]
Hi, I'm at my wit's end as I've seemingly checked and re-checked everything, but still I get no symbols (nor addresses) in the call stack (lib_native.so is my library):

![tL1Lxlyf](https://github.com/user-attachments/assets/b2dd7998-6528-44c0-9d77-89f2dbde30c9)

This started happening some 6-12 months ago, I didn't exactly notice when. Before that it used to work fine. I have re-read the docs and re-done all the setup steps for integrating Crashlytics into my Gradle + Cmake project with a native library.

I have this, of course:
```
firebaseCrashlytics {
    nativeSymbolUploadEnabled true
}
```
and I call `./gradlew uploadCrashlyticsSymbolFileRelease` which returns no errors and takes a while to complete, so it's doing something.

Here is the crashlytic package version, the latest for all I know:
```
    implementation(platform("com.google.firebase:firebase-bom:33.1.2"))
    implementation 'com.google.firebase:firebase-crashlytics-ndk'
    implementation 'com.google.firebase:firebase-analytics'
```
I also added this to the root-level CmakeLists.txt: `link_libraries("-Wl,--build-id")`

Any tips on what could be causing this, what I might be missing?

**Comment by yusufspektra** [comment]
Any news about that?

**Comment by VioletGiraffe** [comment]
@yusufspektra  No. The older builds from the time when it did work still continue to send reports that show up normally, but new builds are broken and nothing I tried worked. I deleted crashlytics from all my company's projects, it's completely non-functional.
